diff --git a/src/js/init.js b/src/js/init.js
index 895348b..acb74b8 100644
--- a/src/js/init.js
+++ b/src/js/init.js
@@ -25,7 +25,8 @@ const {
     BrowserWindow,
     dialog,
     protocol,
-    ipcMain
+    ipcMain,
+    nativeImage
 } = require('electron');
 const path = require('path');
 const url = require('url');
@@ -76,6 +77,9 @@ let init = function () {
         autoHideMenuBar: true
     });
 
+    if (process.platform === 'linux')
+        win.setIcon(nativeImage.createFromPath(path.join(__dirname, '..', 'icons', 'iconColor.png')));
+
     settingsDb.findOne({
         _id: 'bounds'
     }, function (err, doc) {
