diff --git a/src/js/init.js b/src/js/init.js
index b0ed84a..e775180 100644
--- a/main/src/js/init.js
+++ b/main/src/js/init.js
@@ -25,7 +25,8 @@ const {
     BrowserWindow,
     dialog,
     protocol,
-    ipcMain
+    ipcMain,
+    nativeImage
 } = require('electron');
 const path = require('path');
 const url = require('url');
@@ -68,6 +69,9 @@ let init = function () {
         autoHideMenuBar: true
     });
 
+    if (process.platform === 'linux')
+        win.setIcon(nativeImage.createFromPath(path.join(__dirname, '..', 'icons', 'iconColor.png')));
+
     win.loadURL(url.format({
         pathname: path.join(__dirname, '../index.html'),
         protocol: 'file:',
